metagpt/tools/tool_data_type.py

```diff
@@ -1,26 +1,9 @@
-from enum import Enum
-
 from pydantic import BaseModel
 
 
-class ToolTypeEnum(Enum):
-    EDA = "eda"
-    DATA_PREPROCESS = "data_preprocess"
-    FEATURE_ENGINEERING = "feature_engineering"
-    MODEL_TRAIN = "model_train"
-    MODEL_EVALUATE = "model_evaluate"
-    STABLE_DIFFUSION = "stable_diffusion"
-    IMAGE2WEBPAGE = "image2webpage"
-    WEBSCRAPING = "web_scraping"
-    OTHER = "other"
-
-    def __missing__(self, key):
-        return self.OTHER
-
-
 class ToolType(BaseModel):
```

Comment:
In general, it should be called ToolDef

Reply:
I think this class refers to a tool type, code like `DATA_PREPROCESS  = ToolType(name=..., desc=...)` implies that data preprocess is one of tool types with name ... and desc ..., it looks intuitive